component-library/src/app/pages/select-documentation/select-documentation.component.html

```diff
@@ -0,0 +1,664 @@
+            <img
+              src="assets/img/select-content/select-dont-2.png"
+              alt="{{ 'Select.Alt.Dont2' | translate }}"
+            />
+            <h4 class="emphasis text-underline text-color-red">
+              {{ 'General.DontHeading' | translate }}
+            </h4>
+            <p class="body3 subtext-7">
+              {{ 'Select.Dont2' | translate }}
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="element-wrapper-container">
+        <h3 class="h5 emphasis subtitles">
+          {{ 'General.ErrorsHeading' | translate }}
+        </h3>
+        <div class="row">
+          <div class="col-12 col-md-6">
+            <img
+              src="assets/img/select-content/select-do-3.png"
+              alt="{{ 'Select.Alt.Do3' | translate }}"
+            />
+            <h4 class="emphasis text-underline text-color-green">
+              {{ 'General.DoHeading' | translate }}
+            </h4>
+            <p class="body3 subtext-5">{{ 'Select.Do3' | translate }}</p>
+          </div>
+          <div class="col-12 col-md-6">
+            <img
+              src="assets/img/select-content/select-dont-3.png"
+              alt="{{ 'Select.Alt.Dont3' | translate }}"
+            />
+            <h4 class="emphasis text-underline text-color-red">
+              {{ 'General.DontHeading' | translate }}
+            </h4>
+            <p class="body3 subtext-7">
+              {{ 'Select.Dont3' | translate }}
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="element-wrapper-container">
+        <h3 class="h5 emphasis subtitles">
+          {{ 'Select.DesignGuidelines.SubHeading4' | translate }}
+        </h3>
+        <div class="row">
+          <div class="col-12 col-md-6">
+            <img
+              src="assets/img/select-content/select-do-4.png"
+              alt="{{ 'Select.Alt.Do4' | translate }}"
+            />
+            <h4 class="emphasis text-underline text-color-green">
+              {{ 'General.DoHeading' | translate }}
+            </h4>
+            <p class="body3 subtext-5">{{ 'Select.Do4' | translate }}</p>
+          </div>
+          <div class="col-12 col-md-6">
+            <img
+              src="assets/img/select-content/select-dont-4.png"
+              alt="{{ 'Select.Alt.Dont4' | translate }}"
+            />
+            <h4 class="emphasis text-underline text-color-red">
+              {{ 'General.DontHeading' | translate }}
+            </h4>
+            <p class="body3 subtext-7">
+              {{ 'Select.Dont4' | translate }}
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="element-wrapper-container">
+        <h3 class="h5 emphasis subtitles">
+          {{ 'General.MaxMinWidthHeading' | translate }}
+        </h3>
+        <div class="row">
+          <div [style.max-width]="'100%'">
+            <img
+              src="assets/img/select-content/select-min-max.png"
```

Comment:
This img needs to be updated based on Figma changes for tablet and mobile view.
Take a look at icon-buttonn doc page for reference!

Reply:
we'll update the mobile stuff in the mobile enhancement story to avoid confusion